docs/modules/ROOT/pages/how-tos/opsgenie.adoc

```diff
@@ -0,0 +1,263 @@
+----
+
+To make alerts filterable, we add a number of key-value pairs as `details` and a number of values as `tags`.
+OpsGenie allows filtering alerts both by `tag` and by `details.key` and `details.value`.
+Note that tags must be provided as a single comma-separated string to Alertmanager.
+
+[source,yaml]
+----
+openshift4_monitoring:
+  alertManagerConfig:
+    receivers:
+      - name: opsgenie
+        opsgenie_configs:
+          - details:
+              namespace: '{{- if .CommonLabels.exported_namespace -}}{{- .CommonLabels.exported_namespace -}}{{- else if .CommonLabels.namespace -}}{{- .CommonLabels.namespace -}}{{- end -}}'
+              pod: '{{- if .CommonLabels.pod -}}{{- .CommonLabels.pod -}}{{- end -}}'
+              deployment: '{{- if .CommonLabels.deployment -}}{{- .CommonLabels.deployment -}}{{- end -}}'
+              alertname: '{{ .GroupLabels.alertname }}'
+              cluster_id: '{{ .CommonLabels.cluster_id }}'
+              tenant_id: '{{ .CommonLabels.tenant_id }}'
+              severity: '{{ .GroupLabels.severity }}'
+            tags: '{{ .GroupLabels.tenant_id }},
+              {{ .GroupLabels.cluster_id }},
+              {{ .GroupLabels.severity }},
+              {{ .GroupLabels.alertname }},
+              {{ .GroupLabels.namespace }},
+              {{- if .CommonLabels.exported_namespace -}}{{ .CommonLabels.exported_namespace }},{{- end -}}'
+----
+
+Finally, we need to make sure that the Watchdog alert is sent to OpsGenie as a heartbeat instead of a regular alert.
+To this effect, we configure an additional receiver which sends alerts to the OpsGenie REST API integration.
+In particular, this receiver sends alerts to the heartbeat `ping` endpoint for the heartbeat we've configured.
+If you followed our suggestion and used the Commodore `cluster-id` as the name for the heartbeat the snippet below will work out of the box.
+For this receiver you need to provide the API key of the REST API integration, which should be stored in Vault.
+
+In addition to the receiver, we also add a routing configuration to match alerts which are called `Watchdog` and ensure they're sent to the `heartbeat` receiver with a repeat interval of one minute (60 seconds).
+
+[source,yaml]
+----
+openshift4_monitoring:
+  alertManagerConfig:
+    receivers:
+      - name: heartbeat
+        webhook_configs:
+          - send_resolved: false
+            url: https://api.opsgenie.com/v2/heartbeats/${cluster:name}/ping
+            http_config:
+              basic_auth:
+                password: ?{vaultkv:${cluster:tenant}/${cluster:name}/opsgenie/heartbeat-password}
+    route:
+      routes:
+        - match:
+            alertname: Watchdog
+          repeat_interval: 60s
+          receiver: heartbeat
+----
+
+== Full component configuration
+
+Since we've discussed individual elements of the Alertmanager configuration in the previous section, here's the full, copy-pasteable configuration.
+
+[source,yaml]
```

Comment:
I wonder if and were we could put this or part of it, into our global config hierarchy. This is a good docoumentation to have.

Reply:
Having it in the internal global config hierarchy doesn't help other users of Project Syn, and is therefore out of scope for this PR.